robowflex_ompl/include/robowflex_ompl/ompl_interface.h

```diff
@@ -91,6 +91,8 @@ namespace robowflex
 
             std::unique_ptr<ompl_interface::OMPLInterface> interface_{nullptr};  ///< Planning interface.
             std::vector<std::string> configs_;                                   ///< Planning configurations.
+            bool hybridize_{true};    ///< Whether or not planner should hybridize solutions.
```

Comment:
Do these need default values if they are always overridden by the values in the settings?

Reply:
No it does not, you can only modify them through the [ModelBasedPlanningContext](https://github.com/ros-planning/moveit/blob/master/moveit_planners/ompl/ompl_interface/include/moveit/ompl_interface/model_based_planning_context.h#L267)
and not through the [ompl_interface](https://github.com/ros-planning/moveit/blob/master/moveit_planners/ompl/ompl_interface/include/moveit/ompl_interface/ompl_interface.h#L122) although we should change MoveIt at some point to do so